### PR TITLE
Added compilation instructions to README

### PR DESCRIPTION
**Pull request by Menb10ck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,10 @@ required for the bot to authenticate itself with Discord. Look for
 You can copy your `config.json` in either: 
 - `config/`
 - `$HOME/marabot/config/`
+
+### Compilation and Running
+
+To compile the bot, run `dotnet build` in the project root.
```
Comment (sbzappa) [inline README.md:63]: You could also run `dotnet build -c Release Marabot.sln` if you wanted to compile a release version of the bot. Not sure if it's worth mentioning?

Comment (Menb10ck) [reply README.md:63]: :+1:

File: README.md [hunk 2]
```diff
@@ -57,3 +57,14 @@ required for the bot to authenticate itself with Discord. Look for
...
+
+The executable will be located at `MaraBot/bin/Debug/netcoreapp3.1/MaraBot`.
+A daemon script for Linux is provided at `linux/marabotd`.
+
+You can also build in release mode by running `dotnet build -c Release`.
+The executable will then be located at `MaraBot/bin/Release/netcoreapp3.1/MaraBot`.
+Note that the daemon assumes we have a fresh debug build.
```
Comment (sbzappa) [inline README.md:70]: `linux/marabotd` searches for the release build by default.

Comment (Menb10ck) [reply README.md:70]: Whoops

Comment (Menb10ck) [reply README.md]: Fixed it

**Review by sbzappa — APPROVED** [review]

**Pull request merged**